plugins/woocommerce-admin/client/products/sections/product-shipping-section.tsx

```diff
@@ -317,8 +355,16 @@ export function ProductShippingSection( {
 													'woocommerce'
 												)
 											) }
+											placeholder={ getPlaceholderForVariation(
+												product?.dimensions?.length
+											) }
 											onFocus={ () => {
 												setHighlightSide( 'B' );
+												setValueForVariation(
```

Comment:
Same comment on all dimension fields around focusing and setting values.

Reply:
So we are not going to set any value onFocus right? Please check that once the values are been updated the parent values are ignored in future updates. That is the way `setValueForVariation` is working currently. Do you still want me to remove that behabiour?

Reply:
done -> [0a9a813](https://github.com/woocommerce/woocommerce/pull/36234/commits/0a9a81367f888cdd58e3e8f5d616ada75d5573d9)